fpga/mist/minimig_mist.qsf

```diff
@@ -337,6 +337,7 @@ set_instance_assignment -name CURRENT_STRENGTH_NEW 8MA -to VGA_VS
 
 
 
+set_global_assignment -name SYSTEMVERILOG_FILE ../../rtl/minimig/video_mixer.sv

```

Comment:
It would be nice if this was just plain Verilog, so it would be compatible with all simulators, where SV support is pretty weak. I guess just the packed array needs rewrite.